packages/components/src/components/icon/dashboard/ic-dashboard-all-markets.svg

```diff
@@ -0,0 +1 @@
+<svg width="32" height="32" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M28.447 17.184l-.97-.255-.567-.93h2.25l-.026.675a.548.548 0 01-.687.51zM17.082 29.309l.082-2.68 1.118.034.047-1.564 1.676.05.075-2.458 1.168-1.642 1.134-.524 1.774.502 1.077 1.374 2.112.4s-2.86 5.81-10.263 6.508zm-4.325-5.05l-.893-.2-.794-1.388.893-1.29.772-.425.617.425.2 1.19.098.993-.893.695zm-3.844-2.407l.686.942-.857 1.155-2.482-.685-3.382-3.467c-.257-1.648-.35-3.178-.35-3.178a13.666 13.666 0 012.359-8.272l.728-1.058 1.345.807.496.794 1.39.893.893.596.297.694v.992c0 .695.397 3.87.397 3.87l-.992 1.39-1.29.893s-.298.496-.298.992v1.786l1.06.856zM14.51 3.093s8.094-.77 12.206 5.782l-2.184 1.67-2.57-.128-3.34.257-.9 1.156-1.285 1.285-1.67.128-.642-1.413-.13-1.927-.513-1.799.257-1.799-3.552-1.655s1.352-1.352 4.323-1.557zm1.49-2.177c-4.83 0-9.125 2.273-11.886 5.804a15.014 15.014 0 00-3.18 9.973C1.296 24.7 7.902 31.083 16 31.083c8.33 0 15.083-6.754 15.083-15.084S24.33.916 16 .916z" fill="#85ACAF"/></svg>
```

Comment:
I think we need to change fill color for each of these icons to a variable, so that it would change based on theme

Reply:
Actually in fact all these SVGs were part of the old design, will delete them. I will proceed with just using temporary existing SVGs as Mehrzad has another PR for adding all the required SVGs.